python/tvm/autotvm/tophub.py

```diff
@@ -27,6 +27,7 @@
     'rocm':    "v0.02",
     'opencl':  "v0.02",
     'mali':    "v0.04",
+    'vulkan':  "v0.04",
```

Comment:
Revert this change

Reply:
Add alias
```python
'vulkan': 'opencl'
```
at 
https://github.com/dmlc/tvm/pull/2230/files#diff-cc3ebb4206649fe3e6a20c759e8c6580R42